### ISSUE TITLE
Daemon does not start without passing jobs directory as argument

### ISSUE THREAD

**Issue by seoester** [KillingSpark/restic-cronned#3]
Using restic-cronned version `0.0.2` the following occurs:

```
$ restic-cronned
No directory specified
```

Content of `$HOME/.config/restic-cronned/config`
```json
{
    "JobPath": "$HOME/.config/restic-cronned/jobs",
    "SrvConf": {"Port": ":8080"},
    "LogConf": {
        "LogDir": "$HOME/.cache/restic-cronned",
        "MaxAge": 30,
        "MaxSize": 10
    }
}
```

```
$ ls $HOME/.config/restic-cronned/jobs
HomeBackup.json
HomeForget.json 
```

**Comment by KillingSpark** [comment]
This should be fixed in version 0.0.3

**Comment by seoester** [comment]
Works as expected in 0.0.3, thank you :)
